main.js

```diff
@@ -52,60 +52,114 @@ const DEFAULT_PAGE_SIZE = 64; // Page size for retrieving documents from the cus
 
   const options = program.opts();
 
-  if (options.validate) {
-    await validate(options);
-  } else {
+  const migrator = new MigrationClient({
+    sourceKey: options.source,
+    targetKey: options.target,
+    defaultPageSize: DEFAULT_PAGE_SIZE,
+    maxParallelism: options.parallelism,
+  });
+
+  const collectionList = [];
+  if (options.collection) {
```

Comment:
Maybe in this rework, `options.collection` should be treated as a comma-separated list so users could opt-in certain collections per run.

Reply:
ah, okay that would be good!